assignment-final.Rmd

```diff
@@ -1,5 +1,39 @@
 ---
-title: "Final assignment"
+title: "Final assignment: Scientific report"
+output:
+    html_document:
+        toc: false
 ---
 
-> Uploaded when it is handed out.
+The final assignment is fairly straightforward, but also challenging. You, as a
+team, must write up a scientific report/manuscript. The best way to think about 
+this assignment is to imagine you are a scientist and you are writing a paper
+to publish your research. This is essentially what we expect of you from the
+final report.
+
+So, your report should have at least these sections: "Background and Rationale",
+"Research Objectives and Hypotheses", "Methods" (with "Data Description" and
+"Data Analysis" subsections), "Results", "Discussion", and "Final Conclusion".
+
+For the report, you are expected to:
+
+- Search the previous research and literature on your research questions (this
+doesn't have to be extensive).
+- Have clear and explicit objectives and hypotheses.
+- Adequately describe and properly cite the data source you will analyze.
+- Describe your data analysis in sufficient detail for others to understand what
+you did and why.
+- Show all the results of your pre-planned data analysis and any additional 
+explorations you did.
+- Discuss the meaning of your results and how they fit with the previous
+literature.
+
+The report and associated code is expected to:
+
+- Be entirely reproducible.
+- Have well documented code.
+
+You are also expected to:
+
+- Work well as a team.
+- Use Git and GitHub to show your work and workflow with your team.
```

Comment:
Paste the grading rubric here also.

I think we should mention something in terms of page length and number of figures. Just to give them a ballpark, what do you think is reasonable?

Reply:
hm, didn't think of that. only problem is they might aim for that rather than just write something. But what about... ~2500 words and ~6-8 figures/tables?

Reply:
Yes, we should stress that we want them to focus on the writing and that we are flexible in these requirements. But a good ballpark is what you suggested I think, ~2500 - 3000 words, and ~6-8 figures (but they are free to group figures together in panels etc), this is more to give them an idea than to use as a hard criteria for submission.

Reply:
@joelostblom updated. Let me know if that is fine.

Reply:
👍 Thanks!